pkg/apis/scheduling/v1beta1/types.go

```diff
@@ -362,6 +362,10 @@ type QueueSpec struct {
 	// Parent define the parent of queue
 	// +optional
 	Parent string `json:"parent,omitempty" protobuf:"bytes,8,opt,name=parent"`
+
+	// The deserved quota by resource type. This part of resources are allowed to be shared to other queues and can be reclaimed back.
+	// +optional
+	Deserved v1.ResourceList `json:"deserved,omitempty" protobuf:"bytes,9,opt,name=deserved"`
```

Comment:
The amount of resources configured by the user. This part of resource can be shared with other queues and reclaimed back when needed.

Reply:
done